scripts/push_docs_to_branch.sh

```diff
@@ -14,7 +14,7 @@ set -Eeuo pipefail
 
 TODAY=`date "+%Y%m%d"`
 # we build new versions only for minors and majors
-PATTERN_FOR_NEW_VERSION="^refs/tags/[0-9]+\\.[0-9]+\\.0$"
+PATTERN_FOR_NEW_VERSION="^refs/tags/[0-9]+\\.0\\.0$"
 PATTERN_FOR_MICRO_VERSION="^refs/tags/[0-9]+\\.[0-9]+\\.[1-9][0-9]*$"
```

Comment:
```suggestion
PATTERN_FOR_EXISTING_VERSION="^refs/tags/[0-9]+\\.[0-9]+\\.[0-9]+$"
```

🤕 sorry for not spotting this one earlier... We need to change this pattern to match new minors too, now (e.g. 3.1.0, etc...). I think it's the last change required, I've doubled check the full logic and it seems in order ✅